Gemfile

```diff
@@ -2,7 +2,7 @@
 
 source "https://rubygems.org"
 
-ruby "3.2.2"
+ruby File.readlines(File.join(__dir__ , '.tool-versions')).select{|l| l =~ /ruby/}.first.strip.split.last
```

Comment:
To make it so there's one source of truth for the ruby version.

Reply:
Clever!

Reply:
Apparently in an upcoming Bundler version we can just say `ruby file: '.tool-versions'` and it knows what to do...but not just yet!